### PR TITLE
Remove duplicate files which reside in community/ repo

### PR DESCRIPTION
**Pull request by kashifest**
Removing the files which reside in community/ repo to avoid duplication

### CODE REVIEW

File: ADOPTERS.md
Comment (tuminoid) [inline ADOPTERS.md:1]: Remember to update community meeting notes after this PR merges with new link to ADOPTERS file.

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Comment by tuminoid** [comment]
/hold
until https://github.com/metal3-io/community/pull/4 is merged

**Comment by tuminoid** [comment]
/cc @lentzi90 
for top-level approval.

**Review by lentzi90 — COMMENTED** [review]
/approve

**Comment by kashifest** [comment]
> /hold until [metal3-io/community#4](https://github.com/metal3-io/community/pull/4) is merged

These files are already in community repo.

**Comment by tuminoid** [comment]
> > /hold until [metal3-io/community#4](https://github.com/metal3-io/community/pull/4) is merged
> 
> These files are already in community repo.

Oops, true.  My bad, that is only adding new stuff.
/unhold

**Pull request merged**